.github/workflows/make-live.yml

```diff
@@ -0,0 +1,66 @@
+name: Make Live
+# Make live versions of .Rmd files in training modules
+
+# This workflow only runs when it is manually dispatched
+on:
+  workflow_dispatch:
+    inputs:
+      rendering:
+        description: Should rendering be completed? TRUE or FALSE
+        required: true
+        default: 'FALSE'
+
+jobs:
+  # This workflow contains a single job called "build-all"
+  make-live:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container:
+      image: ccdl/training_dev:latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # get the full repo
+          fetch-depth: 0
+          # use alexslemonade-docs-bot
+          token: ${{ secrets.DOCS_BOT_GITHUB_TOKEN }}
```

Comment:
Do we need this? I don't think we do, and removing the `fetch-depth` will make things go faster.

Reply:
Nah, I don't think we need it.